src/index.ts

```diff
@@ -10,3 +10,4 @@ export { useConditionalEffect } from './useConditionalEffect';
 export { useConditionalUpdateEffect } from './useConditionalUpdateEffect';
 export { useSafeState } from './useSafeState';
 export { useMediatedState } from './useMediatedState';
+export { useDebounceCallback } from './useDebouneCallback';
```

Comment:
Filename has a typo here, should be `useDebounceCallback` 😄

Reply:
yikes %)